src/sage/groups/matrix_gps/finitely_generated_gap.py

```diff
@@ -936,11 +936,11 @@ def _new_invariant_is_linearly_independent(F, invariants):
         sage: gens = [matrix(QQ, [[-1,1],[-1,0]]), matrix(QQ, [[0,1],[1,0]])]
         sage: G = MatrixGroup(gens)
         sage: s = Sequence(G.invariants_of_degree(14))                                  # needs sage.rings.number_field
-        sage: s.coefficient_matrix()[0].rank()                                          # needs sage.rings.number_field
+        sage: s.coefficients_monomials()[0].rank()                                          # needs sage.rings.number_field
```

Comment:
I'd suggest to line up the comments again, it's silly, but I find it visually less destracting.

Reply:
Done !